### ISSUE TITLE
AppSharingBitRateKb and VideoBitRateKb are applicable to Skype for Business Online

### ISSUE THREAD

**Issue by naminakao** [MicrosoftDocs/office-docs-powershell#176]
The parameters, AppSharingBitRateKb and VideoBitRateKb, can be set on SfB online.
Currently Description section says they are not applicable to SfB Online.

Title: Set-CsConferencingPolicy
https://docs.microsoft.com/en-us/powershell/module/skype/set-csconferencingpolicy?view=skype-ps

**Comment by kenwith** [comment]
@islubin - I remember this change awhile back to set some parameters that were not relevant to SfB Online.. do you remember the details?

**Comment by islubin** [comment]
I tried to merge in a change I made a while ago to indicated that in the New- and Set- CsConferencingPolicy cmdlets, those two parameters are now applicable.  For some reason my merge doesn’t seem to be working (see the below question you asked me. ☺ ).  I will try again later today…

**Comment by kenwith** [comment]
@islubin - I see the Applies To for these two parameters now in the parameter. At the bottom of the Description there is a paragraph.. should they be removed from this paragraph below?

_The following parameters are not applicable to Skype for Business Online: ApplicationSharingMode, AppSharingBitRateKb, AsJob, AudioBitRateKb, Description, EnableMultiViewJoin, EnableOnlineMeetingPromptForLyncResources, EnableReliableConferenceDeletion, FileTransferBitRateKb, Force, Identity, Instance, MaxMeetingSize, MaxVideoConferenceResolution, PipelineVariable, Tenant, TotalReceiveVideoBitRateKb, and VideoBitRateKb_

**Comment by islubin** [comment]
I think those updates should be there with my latest change – I see them on github website at least, although not on docs yet.

**Comment by kenwith** [comment]
@islubin - you are exactly right. There is some issue with docs.microsoft.com pulling from GitHub. I will get in contact with that team and figure this out. Thanks for the updates!

**Comment by kenwith** [comment]
Confirmed there was an Update to the GitHub API yesterday ( https://developer.github.com/changes/ ) that broke the sync with docs.microsoft.com. Team that owns the sync code has already updated to match new GitHub API and sync is working again. I pushed a new sync and everything is updated now. Closing this one out. Thank you @naminakao for reporting and @islubin for the quick resolution! The docs and the community love you both for it!
